ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2455,6 +2457,70 @@ periodics:
     - name: github-app-credentials
       secret:
         secretName: openshift-prow-github-app
+- agent: kubernetes
+  annotations:
+    ci.openshift.io/description: This runs the check-gh-automation tool to verify
+      that all repos with tide configured allow access to openshift-merge-bot
+    ci.openshift.io/sop: https://github.com/openshift/release/blob/master/docs/dptp-triage-sop/infrastructure-jobs.md#periodic-check-gh-automation
+  cluster: app.ci
+  cron: 0 3 * * 2
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+    workdir: true
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-check-gh-automation-tide
+  spec:
+    containers:
+    - args:
+      - --config-path=./core-services/prow/02_config/_config.yaml
```

Comment:
WOW, what a list of arguments :)

Reply:
A lot of the `ignore` will be able to be removed on this one since it is smart enough to only check repos with tide configured. I wanted to copy them all over initially though to not cause disruption. I will come back and see what we can prune from here after